plugins/ssh/ssh.plugin.zsh

```diff
@@ -0,0 +1,39 @@
+############################################################
+# Take all host sections in .ssh/config and offer them for
+# completion as hosts (e.g. for ssh, rsync, scp and the like)
+# Filter out wildcard host sections.
+local hosts
```

Comment:
`local` doesn't work at the top level of plugin or theme files (currently). It needs to be wrapped in an anonymous function to provide a scope for it.